src/tests/ftest/util/dmg_utils.py

```diff
@@ -321,6 +321,45 @@ def storage_query_list_pools(self, uuid=None, rank=None, verbose=False):
             ("storage", "query", "list-pools"), uuid=uuid, rank=rank,
             verbose=verbose)
 
+    def storage_identify_device(self, uuid=None,  verbose=False):
```

Comment:
These methods should match the command.  Shouldn't this be `storage_identify_vmd`?

Reply:
Done.